mscore/metaedit.h

```diff
@@ -34,21 +34,27 @@ class Score;
 class MetaEditDialog : public QDialog, public Ui::MetaEditDialog {
       Q_OBJECT
 
-      Score* score;
+      Score* m_score;
 
-      bool dirty;
+      bool m_dirty;
 
-      virtual void hideEvent(QHideEvent*);
+      virtual void hideEvent(QHideEvent*) override;
 
-   private slots:
+      bool isSystemTag(QString tag) const;
+//      bool isTransformedSystemTag(QString tag) const;
+
+      bool save();
+
+private slots:
       void newClicked();
-      void setDirty() { dirty = true; }
+      void setDirty(bool dirty = true);
```

Comment:
shouldn't this me `m_dirty = true` here?

Reply:
Well it sets the member m_dirty, but for someone from the outside, seeing setm_dirty would look bizarre.
But since it's parenthesis and not brackets, I do not see the problem here. Is there something I miss?

Reply:
That ' dirty'  is local to that function and known only inside it, you can set it, but not check it. And no, this `setdirty()` does not set 'm_dirty'